src/app/ui/header/header.component.html

```diff
@@ -20,19 +23,30 @@
             <li class="nav-item">
               <a class="nav-link" routerLink="/courses">Courses</a>
             </li>
-            <!--<li class="nav-item">-->
-              <!--<a class="nav-link" routerLink="/upload">Upload</a>-->
-            <!--</li>-->
             <!--<li class="nav-item">-->
               <!--<a class="nav-link disabled" href="#">ScoreBoard</a>-->
             <!--</li>-->
           </ul>
+
           <form class="form-inline my-2 my-lrg-0">
```

Comment:
This form should only include the search field and search button (other buttons are just links, not part of a form).